crates/ruma-common/src/events/room/redaction.rs

```diff
@@ -153,12 +164,65 @@ impl RoomRedactionEventContent {
         Self::default()
     }
 
-    /// Creates a new `RoomRedactionEventContent` with the given reason.
-    pub fn with_reason(reason: String) -> Self {
-        Self { reason: Some(reason) }
+    /// Creates a `RoomRedactionEventContent` with the required field for room version 11.
+    pub fn v11(redacts: OwnedEventId) -> Self {
```

Comment:
For `m.room.aliases`, we have `new_v1` and `new_v6`. Can you deprecate or remove `new` in favor of a `new_v1` and also rename this one?

```suggestion
    pub fn new_v11(redacts: OwnedEventId) -> Self {
```

Reply:
I renamed it, and added it to the changelog